frontend/src/app/App.tsx

```diff
@@ -92,6 +95,8 @@ const App = () => {
           suffix: data.whoami.suffix,
           email: data.whoami.email,
           isAdmin: data.whoami.isAdmin,
+          type: data.whoami.type,
```

Comment:
Does `type` need to be added to the query on line 29?

Reply:
Probably, since Typescript doesn't know what this object is. I've been wanting to look at what it would take to add `graphql-code-generator` to define TS types for our queries but haven't gotten around to it.

Reply:
good point @jeskew-gov. The backend support for `type` isn't in yet, and is not used in this PR. I will remove it.